backend/de.metas.manufacturing/src/main/java/org/eevolution/model/validator/PP_Order_PostMaterialEvent.java

```diff
@@ -79,16 +81,21 @@ public void fireMaterialEvent_deletedPPOrder(
 			ModelValidator.TIMING_AFTER_COMPLETE,
 			// Note: close is currently handled in MPPOrder.closeIt()
 			// for the other timings, we shall first figure out what we actually want
-			//ModelValidator.TIMING_AFTER_REACTIVATE,
+			ModelValidator.TIMING_AFTER_REACTIVATE,
 			//ModelValidator.TIMING_AFTER_UNCLOSE,
 			//ModelValidator.TIMING_AFTER_VOID
 	})
-	public void postMaterialEvent_ppOrderDocStatusChange(@NonNull final I_PP_Order ppOrderRecord)
+	public void postMaterialEvent_ppOrderDocStatusChange(@NonNull final I_PP_Order ppOrderRecord, @NonNull final DocTimingType docTimingType)
 	{
-		final PPOrderChangedEvent changeEvent = PPOrderChangedEventFactory
+		PPOrderChangedEvent changeEvent = PPOrderChangedEventFactory
 				.newWithPPOrderBeforeChange(ppOrderConverter, ppOrderRecord)
 				.inspectPPOrderAfterChange();
 
+		if (docTimingType == DocTimingType.AFTER_REACTIVATE)
+		{
+			changeEvent = changeEvent.withNewDocStatus(DocStatus.InProgress);
```

Comment:
if the new doctype can't always be added to the event, please add a comment on why that is